### DISCUSSION TITLE
[Suggestion] On signal when ship docked to shipyard

### DISCUSSION THREAD

**Discussion by pinesh** [tsunamayo/Starship-EVO#3982] [Suggestions]
To expand logic potential, a shipyard block should be connectable to logic systems, whereby it would send an *on* signal when a ship is docked, and an *off* signal when a ship isn't.

**Comment by tsunamayo** [comment]
Hi, this will be dealt with the addition of the event command. It will work with all systems, not just the shipyard, do stuff when a given system state change. It will also replace the hotkey gate and the query gate as it will do the save stuff.

**Reply by pinesh** [reply]
This is the first I've heard of this new logic block!

Can't wait to see it hit experimental!
